### PR TITLE
cli: add flag to delete local cache directory

### PR DESCRIPTION
**Pull request by adamdecaf**
This pull request makes the following changes:
* A quick way to delete the cache directory nancy uses. 

I didn't encounter a bug or corrupt cache, but wanted to delete the cache directory to make sure I was pulling the latest index. 

cc @bhamail / @DarthHater

### CODE REVIEW

File: ossindex/ossindex.go [hunk 1]
```diff
@@ -50,6 +50,11 @@ func getDatabaseDirectory() (dbDir string) {
 	return usr.HomeDir + "/.ossindex"
 }
 
+// RemoveCacheDirectory deletes the local database directory.
+func RemoveCacheDirectory() error {
+	return os.RemoveAll(getDatabaseDirectory())
```
Comment (DarthHater) [inline ossindex/ossindex.go]: I'm wondering if there is a better way to do this (basically just flush Badger rather than delete the directory)

Comment (DarthHater) [reply ossindex/ossindex.go]: Wonderbar.

I've been wanting to move away from Badger for a lil bit (it's a bit overkill for what we need, to be honest). You've got a reasonable explanation for now! Thanks for answering!

File: README.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ nancy [options] </path/to/Gopkg.lock>
 nancy [options] </path/to/go.sum>
 
 Options:
+  -clean-cache
+    	Deleted local cache directory
```
Comment (DarthHater) [inline README.md]: Lil typo here, probably should be `Deletes local cache directory`

File: configuration/parse.go [hunk 3]
```diff
@@ -32,6 +34,7 @@ func Parse(args []string) (Configuration, error) {
 	flag.BoolVar(&noColorDeprecated, "noColor", false, "indicate output should not be colorized (deprecated: please use no-color)")
 	flag.BoolVar(&config.Quiet, "quiet", false, "indicate output should contain only packages with vulnerabilities")
 	flag.BoolVar(&config.Version, "version", false, "prints current nancy version")
+	flag.BoolVar(&config.CleanCache, "clean-cache", false, "Deleted local cache directory")
```
Comment (DarthHater) [inline configuration/parse.go]: Lil typo here, probably should be Deletes local cache directory

**Comment by adamdecaf** [comment]
I've updated the `-clean-cache` doc line.

**Pull request merged**